readthedocs/projects/models.py

```diff
@@ -573,6 +573,9 @@ def get_production_media_url(self, type_, version_slug):
 
         return path
 
+    def get_static_path(self):
+        return self.proxied_api_host + '/static/'
```

Comment:
This seems worse than using a setting -- why are we hard coding this on the model?